### DISCUSSION TITLE
Engine image has still 1 reference after upgrade

### DISCUSSION THREAD

**Discussion by tcoupin** [longhorn/longhorn#5412] [Q&A]
Hi!

I've upgraded to 1.4.0 and used the automated engine upgrade but the old engine image 1.3.2 has still 1 reference and I can not delete it.
How can I have the list of the referencer object ??

<img width="1658" alt="image" src="https://user-images.githubusercontent.com/7457386/219847428-463d8c47-a9db-44ae-ac52-567cdbd17046.png">

**Comment by derekbit** [comment]
Can you provide the support bundle?

**Reply by tcoupin** [reply]
[supportbundle_3443fc10-1b09-4369-a7f9-ab8b5597f61a_2023-02-20T10-02-11Z.zip](https://github.com/longhorn/longhorn/files/10783237/supportbundle_3443fc10-1b09-4369-a7f9-ab8b5597f61a_2023-02-20T10-02-11Z.zip)

**Reply by weizhe0422** [reply]
It appears that one of the replicas in volume `pvc-bac3645d-f072-469b-aae0-294ce2078ce7` was not successfully upgraded to v1.4.0.
![Screenshot from 2023-02-20 21-56-45](https://user-images.githubusercontent.com/17548901/220127118-357a1646-dba2-4a17-887a-fc3d084a0b03.png)

I think you can refer to [here](https://longhorn.io/docs/1.4.0/deploy/upgrade/upgrade-engine/) to upgrade the longhorn-engine to v1.4.0

**Comment by tcoupin** [comment]
Thank you so much!

![image](https://user-images.githubusercontent.com/7457386/220140943-708c56c8-93e3-4bf1-92ae-72d96ec05b91.png)


There are already 3 replicas, so I delete it
